pkg/tokens/builtin/server/dataplane_token_server.go

```diff
@@ -0,0 +1,136 @@
+package server
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/Kong/kuma/pkg/core"
+	"github.com/Kong/kuma/pkg/core/xds"
+	"github.com/Kong/kuma/pkg/tokens/builtin"
+	"github.com/Kong/kuma/pkg/tokens/builtin/issuer"
+	"github.com/pkg/errors"
+	"io/ioutil"
+	"net/http"
+
+	config_core "github.com/Kong/kuma/pkg/config/core"
+	core_runtime "github.com/Kong/kuma/pkg/core/runtime"
+)
+
+func SetupServer(rt core_runtime.Runtime) error {
+	switch env := rt.Config().Environment; env {
+	case config_core.KubernetesEnvironment:
+		return nil
+	case config_core.UniversalEnvironment:
+		generator, err := builtin.NewDataplaneTokenIssuer(rt)
+		if err != nil {
+			return err
+		}
+		srv := &DataplaneTokenServer{
+			Port:   rt.Config().DataplaneTokenServer.Port,
+			Issuer: generator,
+		}
+		if err := core_runtime.Add(rt, srv); err != nil {
+			return err
+		}
+	default:
+		return errors.Errorf("unknown environment type %s", env)
+	}
+	if rt.Config().Environment != config_core.KubernetesEnvironment {
+	}
```

Comment:
What is the purpose of these lines ?

Reply:
I think I did a mistake when merging/rebasing. Fixed